cli.go

```diff
@@ -68,6 +69,10 @@ const (
 	envDebug = "TGF_DEBUG"
 )
 
+const autoUpdateFileName = "auto-update"
+
+var dueForUpdate = lastRefresh(autoUpdateFileName) > 2*time.Hour
```

Comment:
This should still go in the method though. You want to call this right before the `if`